### PR TITLE
[QOL] Better Visual of Bridge Indicator

### PR DESCRIPTION
**Pull request by zethnest**
This is how the bridge indicator looks like.

### CODE REVIEW

File: core/src/mindustry/world/blocks/distribution/ItemBridge.java [hunk 1]
```diff
@@ -167,6 +167,37 @@ public void drawConfigure(){
...
+                    if(!linked && !(other.<ItemBridgeEntity>ent().link == tile.pos())) continue;
+                    float tx = tile.drawx(), ty = tile.drawy(), ox = other.drawx(), oy = other.drawy();
+                    float alpha = Math.abs((linked ? 100 : 0)-(Time.time() * 2) % 100f) / 100;
+                    float x = Interpolation.linear.apply(ox, tx, alpha);
+                    float y = Interpolation.linear.apply(oy, ty, alpha);
+
+                    Tile link = linked ? tile : other;
+                    Tile otherLink = linked ? other : tile;
+
+                    Draw.color(Color.valueOf("444444"));
```
Comment (Anuken) [inline core/src/mindustry/world/blocks/distribution/ItemBridge.java:186]: `Color.valueOf` creates garbage. Cache this as a final static variable.

File: core/src/mindustry/world/blocks/distribution/ItemBridge.java [hunk 2]
```diff
@@ -167,6 +167,37 @@ public void drawConfigure(){
...
+                    Lines.stroke(2.5f);
+                    Lines.circle(ox, oy, 2f);
```
Comment (Anuken) [inline core/src/mindustry/world/blocks/distribution/ItemBridge.java:188]: Circles don't really fit here, and may reduce performance. Draw a 45° rotated square instead (`Lines.poly()`)

File: core/src/mindustry/world/blocks/distribution/ItemBridge.java (hunk 1 above)
Comment (zethnest) [reply core/src/mindustry/world/blocks/distribution/ItemBridge.java]: I found that `Pal.gray` is very close to `Color.valueOf("444444")` so I used that.

**Comment by zethnest** [comment]
gifine
This is what it currently looks like. I've tried using it in 5.0, and it doesn't impede or even disrupt what I'm doing at the moment at all. It sometimes helps me in identifying which goes to which.

**Comment by Anuken** [comment]
Merged in 2280ed6

**Pull request closed without merging**